input.scss

```diff
@@ -0,0 +1,37 @@
+@use 'node_modules/@mapsindoors/midt/font';
+@use 'node_modules/@mapsindoors/midt/border';
+@use 'node_modules/@mapsindoors/midt/padding';
+@use 'node_modules/@mapsindoors/midt/margin';
+@use 'node_modules/@mapsindoors/midt/opacity';
+@use 'node_modules/@mapsindoors/midt/sizing';
+
+.mi-input {
+
+    &[type="text"],
+    &[type="number"] {
+        @include font.size(medium);
+        @include padding.all(x-small);
+        @include border.style(solid);
+        @include border.width(medium);
+        @include border.gray(60);
+        @include border.radius(medium);
+    }
+
+    &[type="text"],
+    &[type="number"],
+    &[type="range"] {
+        display: block;
+    }
+
+    &[type="radio"],
+    &[type="checkbox"] {
+        margin: 0px;
+        -webkit-transform: scale(1.2,1.2);
```

Comment:
I am not sure this one with a prefix should be necessary. Or is it for Safari?

Reply:
> _"Most major browsers now support the official CSS3 transform property. Unless you need to support a really old browser, there's no need to use the -webkit- extension."_

Removed 👍